src/main/java/it/gov/pagopa/payhub/activities/activity/debtposition/ValidateDebtPositionActivityImpl.java

```diff
@@ -0,0 +1,152 @@
+package it.gov.pagopa.payhub.activities.activity.debtposition;
+
+import it.gov.pagopa.payhub.activities.dao.IngestionFlowFileDao;
+import it.gov.pagopa.payhub.activities.dao.TaxonomyDao;
+import it.gov.pagopa.payhub.activities.dto.IngestionFlowFileDTO;
+import it.gov.pagopa.payhub.activities.dto.PersonDTO;
+import it.gov.pagopa.payhub.activities.dto.TransferDTO;
+import it.gov.pagopa.payhub.activities.dto.debtposition.DebtPositionDTO;
+import it.gov.pagopa.payhub.activities.dto.debtposition.DebtPositionTypeOrgDTO;
+import it.gov.pagopa.payhub.activities.dto.debtposition.InstallmentDTO;
+import it.gov.pagopa.payhub.activities.dto.debtposition.PaymentOptionDTO;
+import it.gov.pagopa.payhub.activities.exception.ValidationException;
+import it.gov.pagopa.payhub.activities.utility.Utilities;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+import org.springframework.util.CollectionUtils;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import static it.gov.pagopa.payhub.activities.utility.Utilities.isValidIban;
+import static it.gov.pagopa.payhub.activities.utility.Utilities.isValidPIVA;
+
+@Lazy
+@Service
+public class ValidateDebtPositionActivityImpl implements ValidateDebtPositionActivity {
+
+    private final IngestionFlowFileDao ingestionFlowFileDao;
+    private final TaxonomyDao taxonomyDao;
+
+    public ValidateDebtPositionActivityImpl(IngestionFlowFileDao ingestionFlowFileDao, TaxonomyDao taxonomyDao) {
+        this.ingestionFlowFileDao = ingestionFlowFileDao;
+        this.taxonomyDao = taxonomyDao;
+    }
+
+    public void validate(DebtPositionDTO debtPositionDTO) {
+        List<IngestionFlowFileDTO> flows = ingestionFlowFileDao.getIngestionFlowFilesByOrganization(
+                debtPositionDTO.getOrg().getOrgId(), true);
+
+        if (CollectionUtils.isEmpty(flows)) {
+            throw new ValidationException("No flow was found for organization with id " + debtPositionDTO.getOrg().getOrgId());
```

Comment:
@serdimic why is it required to have an IngestionFlowFile when creating manually a debtPosition?

CC: @robertopasetti

Reply:
it's probably a bad design choice just inherited from MyPay3, where any debt position(dovuto) has a N:1 relation with a Flow. For handling debt position created manually (from operator webapp view "insert debt position"; from WS "paaSilImportaDovuto" or "paaSilInviaDovuti"; from spontaneous payment), fake virtual flows are defined on any organization.
I would not replicate this design choice on new data model, leaving the field flow optional, and only used when the debt position is created by a massive flow.

Reply:
I agree @serdimic do not replicate something it's not necessary anymore.

Reply:
ok, @Benedetta-fabbri  let's keep these lines on MyPay code, but let's remove them from here

Reply:
I'll remove this check